docs/sources/reference/commandline/cli.md

```diff
@@ -1100,6 +1100,10 @@ and Docker images will report:
 
     untag, delete
 
+The `--since` and `--until` parameters can be Unix timestamps, RFC3339
+dates or Go duration strings (e.g. `10m`, `1h30m`) computed relative to
```

Comment:
Hm not sure Wdyt, would a regular Docker user find it useful? The "valid units" might be something, but I doubt someone will use `ns` or `us` :)